docker-base-runtime/Dockerfile

```diff
@@ -1,13 +1,13 @@
 # A temporary stage just to compile schedrr
-FROM ubuntu:bionic-20190807 as build-schedrr
+FROM ubuntu:bionic-20191202 as build-schedrr
```

Comment:
Sure it is pretty benign, but shouldn't we keep these two changes (strip Python2 and switch base image) distinct to avoid conflating issues between the two ?

Reply:
Fair enough. I've split that into a separate PR (#43) and rewritten it out of the history of this one.